packages/app/src/modules/options/scatterConfig.js

```diff
@@ -0,0 +1,90 @@
+import AxisTitle from '../../components/VisualizationOptions/Options/AxisTitle'
+import AxisRange from '../../components/VisualizationOptions/Options/AxisRange'
+import AxisSteps from '../../components/VisualizationOptions/Options/AxisSteps'
+import AxisDecimals from '../../components/VisualizationOptions/Options/AxisDecimals'
+import AxisLabels from '../../components/VisualizationOptions/Options/AxisLabels'
+import TargetLine from '../../components/VisualizationOptions/Options/TargetLine'
+import BaseLine from '../../components/VisualizationOptions/Options/BaseLine'
+import getDataTab from './tabs/data'
+import getAdvancedSection from './sections/advanced'
+import getDisplayTemplate from './sections/templates/display'
+import getLinesTemplate from './sections/templates/lines'
+import InfoText from '../../components/VisualizationOptions/InfoText'
+
+const verticalAxisId = 'RANGE_0'
+const horisontalAxisId = 'RANGE_1'
+
+export default () => [
+    getDataTab([
+        getDisplayTemplate({
+            content: React.Children.toArray([<SkipRounding />]),
+        }),
+        getLinesTemplate({
+            // TODO: Add trend line?
+            content: React.Children.toArray([
+                <InfoText
+                    text={i18n.t(
+                        'Base and target lines are available on the Axes tab for scatter charts'
+                    )}
+                />,
+            ]),
+        }),
+        getAdvancedSection(),
+    ]),
+    getAxesTab([
+        {
+            key: 'axes-vertical-axis',
+            label: i18n.t('Vertical axis'),
+            content: React.Children.toArray([
+                <AxisTitle
+                    axisId={verticalAxisId}
+                    fontStyleKey={FONT_STYLE_VERTICAL_AXIS_TITLE}
+                />,
+                <AxisRange axisId={verticalAxisId} />,
+                <AxisSteps axisId={verticalAxisId} />,
+                <AxisDecimals axisId={verticalAxisId} />,
+                <AxisLabels axisId={verticalAxisId} />,
+                <TargetLine axisId={verticalAxisId} />,
+                <BaseLine axisId={verticalAxisId} />,
+            ]),
+        },
+        {
+            key: 'axes-horizontal-axis',
+            label: i18n.t('Horizontal axis'),
+            content: React.Children.toArray([
+                <AxisTitle
+                    axisId={horisontalAxisId}
+                    fontStyleKey={FONT_STYLE_HORIZONTAL_AXIS_TITLE}
+                />,
+                <AxisRange axisId={horisontalAxisId} />,
+                <AxisSteps axisId={horisontalAxisId} />,
+                <AxisDecimals axisId={horisontalAxisId} />,
+                <AxisLabels axisId={horisontalAxisId} />,
+                <TargetLine axisId={horisontalAxisId} isVertical={true} />,
+                <BaseLine axisId={horisontalAxisId} isVertical={true} />,
+            ]),
+        },
+    ]),
+    getSeriesTab(),
+    getStyleTab([getTitlesSection()]),
+    getOutliersTab([
```

Comment:
Does this resolve to an empty tab?
Should it be removed until specific options for outliers are available?

Reply:
Yes, I added an empty tab in hope of outliers being part of this PR, but as the PR grew too large and outliers aren't ready yet it'll come in a later PR instead. As noted in the **Known issues** in the description above, 

> This PR primarily focuses on the Axis tab in the Options modal, so the rest of the tabs will have to be revisited in a future PR.